vulcan/hebe/_hebe_data.py

```diff
@@ -61,3 +61,15 @@ async def get_exams(
         :rtype: Union[AsyncIterator[:class:`~vulcan.hebe.data.Exam`], List[int]]
         """
         return Exam.get(self._api, last_sync, deleted, **kwargs)
+
+    async def get_homeworks(
```

Comment:
There isn't any plural form of "homework". Instead, use just "homework" or "homework list". (In this line it's better to use just "homework")